crates/oxc_linter/src/rules/jsx_a11y/anchor_has_content.rs

```diff
@@ -114,12 +146,27 @@ fn test() {
         (r"<a title={title} />", None, None),
         (r"<a aria-label={ariaLabel} />", None, None),
         (r"<a title={title} aria-label={ariaLabel} />", None, None),
+        (r#"<a><Bar aria-hidden="false" /></a>"#, None, None),
+        // anchors can be hidden
+        (r"<a aria-hidden>Foo</a>", None, None),
+        (r#"<a aria-hidden="true">Foo</a>"#, None, None),
+        (r"<a hidden>Foo</a>", None, None),
+        (r"<a aria-hidden><span aria-hidden>Foo</span></a>", None, None),
+        (r#"<a hidden="true">Foo</a>"#, None, None),
+        (r#"<a hidden="">Foo</a>"#, None, None),
+        // TODO: should these be failing?
+        (r"<a><Bar hidden /></a>", None, None),
+        (r#"<a><Bar hidden="" /></a>"#, None, None),
+        (r#"<a><Bar hidden="until-hidden" /></a>"#, None, None),
```

Comment:
good question 🙂 probably not? if they are capitalized, that infers that it's a react component rather than a html one. so should be allowed?

Reply:
> good question 🙂 probably not? if they are capitalized, that infers that it's a react component rather than a html one. so should be allowed?

It has less to do with `Bar` and more to do with `hidden`. `aria-hidden` gets reported, but `hidden` does not.